README.md

```diff
@@ -41,6 +43,8 @@ iwr https://win.wasmer.io -useb | iex
 cd as_abi_protobuf && npm install
 
 cargo install fd-find
+cargo install --force cargo-make
```

Comment:
only needed for massa-rust-sc-examples (postponed)
I did not have the time to test if cargo xtask is best suited
would you mind adding this as a comment?

Reply:
Sure, updating rn